scripts/superdesk-desks/views/desk-config-modal.html

```diff
@@ -40,7 +40,7 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                             <div class="field desk-type">
                                 <label translate>Desk Type</label>
                                 <select id="deskType" ng-model="desk.edit.desk_type"
-                                        ng-options="deskType.value as deskType.name for deskType in deskTypes"
+                                        ng-options="type.qcode as type.name for type in deskTypes"
```

Comment:
It looks like the vocabulary still has name and value not qcode as attributes: 
https://github.com/superdesk/superdesk-core/blob/merge-pa/apps/prepopulate/data_initialization/vocabularies.json#L272
But maybe I am missing something.